Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 
 Compiler Features:
+ * Commandline Interface: Add ``--asm-json`` output in assembler mode.
```

Comment:
Some entries for new options have a simple line explainer, while some others don't. So suggesting that one be added, but also fine otherwise.